### ISSUE TITLE
v7.0.109 suspending tabs despite activity

### ISSUE THREAD

**Issue by Anyasia** [deanoemcke/thegreatsuspender#739]
* Extension version: v7.0.109
 * Browser & version: Chrome 68.0.3440.106
 * Operating system & version: Windows 8.1

Since the latest update, tabs seem to be being suspended more often than they should be. Specifically, it appears as though very brief segments of activity on a tab (less than ~1 second) do not register as the tab being active, and as such the tab suspends in spite of it having been recently active. I've most commonly experienced this while `Ctrl+Tab`ing through tabs without spending significant time on them.

Because of the above-described behavior, my _guess_ is that this was intentionally-implemented under the assumption that if a user merely briefly tabs through a tab, he would not want that to count as 'activity.' Unfortunately, this eliminates the very useful tactic of scrolling through multiple open tabs to specifically mark them as active, and therefore prevent their suspension, without having to individually mark each one as 'don't suspend for now'—or in cases where one _wants_ them to suspend eventually, but just… not right now.

This would be far-better-implemented as an option, rather than simply making the assumption that users don't want such activity to count as such. I'm not even sure how long I've used this extension, but suffice it to say this is a use case I've grown quite accustomed to, and it's rather annoying to have tabs arbitrarily suspend when I know I was just on them.

**Comment by deanoemcke** [comment]
@Anyasia as you assumed, it was an intentional change. although i'm pretty sure that same behaviour existed in the previous release also.
rather than rely on a trick of quickly visiting a bunch of tabs, i'd rather implement a more obvious way for users to achieve the same task.

would an option to be able to _pause_ the extension be what you're looking for? this is already a feature request which i am considering: https://github.com/deanoemcke/thegreatsuspender/issues/256

either that, or just extending the time to suspend might help you?

**Comment by Anyasia** [comment]
@deanoemcke That's very odd that you say it existed in the previous release, since I've only noticed it as a problem since the update. Given _how_ noticeable a change it was, I even wonder if I'd previously somehow missed an update and was still using a much older version.

Pausing the extension would certainly be a welcome feature, and would mitigate some of my concern with the behavior. Still going to be a usage pattern change to relearn, but given I just noticed this latest release also removed the 'don't suspend for now' option from the extension dropdown anyway, it seems that I shall have to be doing that anyway! ;)

Thanks again for your attention to this matter.
